presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1522,29 +1522,35 @@ public void testBucketedExecution()
     @Test
     public void testScaleWriters()
     {
-        // small table that will only have one writer
-        assertUpdate(
-                Session.builder(getSession())
-                        .setSystemProperty("scale_writers", "true")
-                        .setSystemProperty("writer_min_size", "32MB")
-                        .build(),
-                "CREATE TABLE scale_writers_small AS SELECT * FROM tpch.tiny.orders",
-                (long) computeActual("SELECT count(*) FROM tpch.tiny.orders").getOnlyValue());
+        try {
```

Comment:
Dear Reviewer,

believe it or not, the only change here is wrapping in try-finally (adding the finally block)